app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabViewHolder.kt

```diff
@@ -40,11 +41,15 @@ class RecentTabViewHolder(
     @Composable
     override fun Content() {
         val recentTabs = components.appStore.observeAsComposableState { state -> state.recentTabs }
+        val recentSyncedTabState = components.appStore.observeAsComposableState { state -> state.recentSyncedTabState }
 
         RecentTabs(
             recentTabs = recentTabs.value ?: emptyList(),
+            recentSyncedTabState = recentSyncedTabState.value ?: RecentSyncedTabState.None,
```

Comment:
I am starting to think we should just rip out the Recent Synced Tabs into its own component separate from `RecentTabs` and do the following:

```
Column {
  RecentTabs()
  Spacer()
  RecentSyncedTab()
}
```
This has the benefit of separating out the state and callback parameters of 2 very distinct components. I don't think Layout would be impeded either.

Reply:
I separated them into two separate composables. Were you suggesting to also move the synced tab stuff out of `RecentTabController` and into its own handler?

Reply:
I will leave that up to you. I am on the fence since there's a `RecentTab.SyncedTab` so maybe I'll draw your attention towards whether or not `SyncedTab` should belong in the sealed class of `RecentTab`. Before I think it made sense assuming we were appending to the `List<RecentTab>` and therefore determining whether or not we would render the component based on it, but if this is gonna be a totally unrelated state, maybe we should also move it out.